app/src/main/java/com/duckduckgo/app/browser/DuckDuckGoWebView.kt

```diff
@@ -227,7 +227,9 @@ class DuckDuckGoWebView : WebView, NestedScrollingChild3 {
 
     @SuppressLint("ClickableViewAccessibility")
     override fun onTouchEvent(ev: MotionEvent): Boolean {
-        var returnValue = false
+        parent.requestDisallowInterceptTouchEvent(true)
```

Comment:
why?

Reply:
Prevents the ViewPager swiping on the WebView.